src/ontology/cl-edit.owl

```diff
@@ -21080,9 +21080,9 @@ AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_0004120 "https
 AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_0004120 "https://orcid.org/0000-0002-5260-9315")
 AnnotationAssertion(oboInOwl:hasBroadSynonym obo:CL_0004120 "type I")
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0004120 "BAMS:1045")
-AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0004120 "I-2")
 AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0004120 "inner alpha retinal ganglion cell")
 AnnotationAssertion(oboInOwl:hasOBONamespace obo:CL_0004120 "cell")
+AnnotationAssertion(oboInOwl:hasRelatedSynonym obo:CL_0004120 "I-2")
```

Comment:
Consider adding abbreviation tag to "I-2".

Reply:
In this case I was not sure if it was an abbreviation, as the label or the other sinonyms have nothing to do with it.